src/verifiers.rs

```diff
@@ -5,9 +5,9 @@
 
 use std::borrow::Borrow;
 use std::collections::HashMap;
-// use std::hash::Hash;  // Hasher
 
-use pairing::{CurveAffine, CurveProjective};  // Engine, Field, PrimeField, SqrtField
+use ark_ec::ProjectiveCurve;
+use ark_serialize::{CanonicalSerialize};
 
 use super::*;
 
```

Comment:
We'll keep these I guess, and they could be useful if collecting by gossip, but we do not envision using them ourselves, and maybe https://eprint.iacr.org/2021/005 provides interesting ides for gossip situations.

Reply:
Hmm, what do you mean by "these"?

Reply:
I meant the whole file with these aggregate BLS verifiers.